### PR TITLE
Feature/test logic

### PR DESCRIPTION
**Pull request by fitodic**
Two new demo apps were added (empty and clean) to test all possibilites.

### CODE REVIEW

File: demo/empty/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (mislavcimpersak) [inline demo/empty/admin.py]: delete unnecessary files such as models.py and admin.py

File: unload/logic.py [hunk 2]
```diff
@@ -4,64 +4,48 @@
 
 import sys
 
-from django.template.backends.django import DjangoTemplates
-from django.conf import settings
-
 from .base import Template
-from .utils import (get_contents, get_package_locations,
-                    get_template_files, output_as_table, output_template_name)
+from .utils import (get_app, get_contents, get_package_locations,
```
Comment (mislavcimpersak) [inline unload/logic.py]: reformat this as one import per line.

File: .travis.yml [hunk 3]
```diff
@@ -23,6 +23,7 @@ install:
   - pip install tabulate
   - pip install coveralls
   - pip install tox
+  - pip install Jinja2
```
Comment (mislavcimpersak) [inline .travis.yml]: get rid of Jinja2 stuff as discussed

**Comment by mislavcimpersak** [comment]
also if `db.sqlite3` is not used, remove it. there is no need for db file in repo

File: unload/tests/test_utils.py [hunk 4]
```diff
@@ -12,7 +12,8 @@
 
 from ..utils import (get_app, get_contents, get_filters, get_package_locations,
                      get_template_files, get_templatetag_members,
-                     output_template_name, output_as_table, update_dictionary)
+                     output_template_name, output_as_table, update_dictionary,
```
Comment (mislavcimpersak) [inline unload/tests/test_utils.py]: also here - one import per line as explained

**Comment by fitodic** [comment]
Changes uploaded

**Comment by mislavcimpersak** [comment]
please remove sqlite file too

**Pull request merged**